src/files/ThemeXml.ts

```diff
@@ -0,0 +1,149 @@
+import { Archive } from '../classes/Archive.ts';
+import { XmlFile } from '../classes/XmlFile.ts';
+import { FileMime } from '../enums.ts';
+import { QNS } from '../utilities/namespaces.ts';
+import { evaluateXPathToMap } from '../utilities/xquery.ts';
+import { create } from '../utilities/dom.ts';
+
+/**
+ * Represents the various 'scheme' elements that comprise a theme.
+ *
+ * Currently, only FontScheme is implemented. It functions as a fallback for
+ * determining which fonts to apply when the StylesXml cannot determine which
+ * font to apply for the required Normal style in MS Word.
+ *
+ * @TODO Implement ColorScheme and FormatScheme
+ */
+export type ThemeElements = {
+	fontScheme: FontScheme;
+}
+
+export type FontScheme = {
+	name: string;
```

Comment:
What's `name` used for? How can someone who creates an `ThemeXml` instance manually (instead of importing it from OOXML) determine a good name?

Or even better - can docxml generate one in that case?

Reply:
As far as I can tell, it's only used as the name displayed to users in Word.